framework/midi/midievent.h

```diff
@@ -985,13 +985,13 @@ struct Event {
         // simple bit shift
         size_t scaleBits = dstBits - srcBits;
         uint32_t bitShiftedValue = srcVal << scaleBits;
-        uint32_t srcCenter = 2 ^ (srcBits - 1);
+        uint32_t srcCenter = uint32_t(2 ^ (srcBits - 1));
         if (srcVal <= srcCenter) {
             return bitShiftedValue;
         }
         // expanded bit repeat scheme
         size_t repeatBits = srcBits - 1;
-        uint32_t repeatMask = (2 ^ repeatBits) - 1;
+        uint32_t repeatMask = uint32_t((2 ^ repeatBits) - 1);
```

Comment:
use static_cast instead of old C-style cast

Reply:
dito